src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -88,6 +89,15 @@ def get(self, request: Request, project) -> Response:
                     and not has_latest_adopted_release
                 ):
                     continue
+                if (
+                    context["id"] == "sentry.rules.filters.tagged_event.TaggedEventFilter"
+                    or context["id"] == "sentry.rules.filters.event_attribute.EventAttributeFilter"
+                ) and not has_is_in:
+                    # Filter the form_fields to strip `is_in` choices
+                    match_choices = context["formFields"]["match"]["choices"]
+                    context["formFields"]["match"]["choices"] = [
```

Comment:
hmmmmmm....

Ok - so in the tests this is failing because it seems like this is modifying the global registry for `rules` and i'm not entirley clear if htis is a test issue or whether this would actually affect the host registry or something 🤔 

not entirley sure how i would validate this...

@wedamija would you have any idea?

Reply:
oof deepcopy 🙏 👍